cmd/web-handlers.go

```diff
@@ -546,10 +546,15 @@ func (web *webAPIHandlers) Download(w http.ResponseWriter, r *http.Request) {
 	objectLock.RLock()
 	defer objectLock.RUnlock()
 
-	if err := objectAPI.GetObject(bucket, object, 0, -1, w); err != nil {
+	info, err := objectAPI.GetObjectInfo(bucket, object)
```

Comment:
You forgot to remove GetObjectInfo() usage @poornas , revert the changes for web-handlers.go